hathor/builder/cli_builder.py

```diff
@@ -49,9 +49,10 @@
 
 
 class SyncChoice(Enum):
-    V1_ONLY = auto()
-    V2_ONLY = auto()
-    BRIDGE = auto()
+    V1_DEFAULT = auto()  # v2 can be enabled in runtime
```

Comment:
Maybe `v1 enabled, v2 disabled but can be enabled in runtime`?

Reply:
Done.